fixtures/anonymous_class_declaration.cpp.native

```diff
@@ -46,19 +37,6 @@
                IsFrozen: true,
                LocOffsetEnd: 18,
                LocOffsetStart: 9,
-               'Prop_ImplicitNames': [
```

Comment:
Node disappeared

Reply:
Yes, good catch, disabled them while debugging some stuff and forgot to re-enable again. Done.